### ISSUE TITLE
Tusk Not Closing KDBX File - Always Reads from cache

### ISSUE THREAD

**Issue by hexcalibr** [subdavis/Tusk#106]
## This issue is a 

- [x] bug
- [ ] feature
- [ ] question

### Please describe the current behavior, and explain why it's bad.

Updated or new entries in cloud-stored file are not being read by Tusk (Chrome v64/Windows 10)

### Please describe how you think it should change.

When I close my kdbx file in Tusk and reopen it, I would expect that Tusk re-read the file thereby always having the latest information in the file available to me.

For example, I've updated a few entries since switching to Tusk, and added new entries. If I use Tusk to copy credentials from the Google Drive stored file, even after closing the file and re-opening,  still see the old credentials and do not see any new credentials.

**Comment by subdavis** [comment]
There's a 60 second cache of the current kdbxweb file.  If you close and immediately open, you are seeing this cache.  You can force a cache reload by locking the database manually and re-entering your password.  I see that you say you've tried this, so that part might be a bug and I'll look into it.

FYI: in the "how you think it should change", you didn't say how you think it should change :P

**Comment by subdavis** [comment]
Question: do your changes ever show up?  How long does it take for this to happen?  Which cloud storage provider are you using?

**Comment by hexcalibr** [comment]
I'll try closing and waiting for 60+ seconds to see if my updates come through...

**Comment by hexcalibr** [comment]
I'm using Google to store the file.
I've tried closing and waiting for over a minute, but no good.
I've also tried copying the file and opening the copy (which has a different name) and no go.

Should I expect both of the Delete buttons in Tusk's Advanced Settings tab to clear the stored object data?

**Comment by hexcalibr** [comment]
@subdavis I think I have found the issue... And it's not with Tusk!

I use Keepass2Android primarily, which as you probably know allows one to edit your database. I'm _sure_ that I used to be able to make a change, sync the file, then see the updates on whatever Chrome kdbx reader I was using at the time. This seems to no longer be the case.

At least on my phone (Moto G5s) with the Android version I'm using (7.1.1) and Keepass2Android v1.04b (just updated last night too) users must export the database to their cloud storage. The message that told me the remote and local file are synced is not reliable.
